openedx/core/djangoapps/user_api/models.py

```diff
@@ -27,6 +30,10 @@ class UserPreference(models.Model):
     key = models.CharField(max_length=255, db_index=True, validators=[RegexValidator(KEY_REGEX)])
     value = models.TextField()
 
+    TIME_ZONE_CHOICES = [
+        [tz, get_formatted_time_zone(tz)] for tz in common_timezones
```

Comment:
`get_formatted_time_zone()` keeps the time zone name at the beginning of the string so it should be alphabetical since `common_timezones` is sorted alphabetically